addons/interaction/functions/fnc_manageAce3Interactions.sqf

```diff
@@ -0,0 +1,96 @@
+/**
+ * Receives a status update and sets ACE3 dragging, carrying and cargo accordingly.
+ *
+ * Arguments:
+ * 0: Equipment <OBJECT>
+ * 1: Condition <STRING>
+ * 2: Status <BOOLEAN>
+ *
+ * Returns:
+ * none
+ *
+ * Example:
+ * [_computer, "powerConnected", true] call AE3_interaction_fnc_manageAce3Interactions
+ */
+
+params ["_target", "_condition", "_status"];
+
+// ace_dragging_fnc_setCarryable and ace_dragging_fnc_setDraggable need to be executed on all machines with JIP
+// ace_cargo_fnc_setSize can be executed locally because it handles JIP and remoteExecCall on it's own
+
+//this function only runs once per triggered interaction by player on players machine
+private _settingsAce3 = _target getVariable "AE3_SettingsACE3";
+if (!isNil "_settingsAce3") then
+{
+    if (!(_condition isEqualTo "init")) then { _settingsAce3 set [_condition, _status]; };
+
+    private _powerConnected = _settingsAce3 getOrDefault ["powerConnected", false, true];
+    private _networkConnected = _settingsAce3 getOrDefault ["networkConnected", false, true];
+    private _unwrapped = _settingsAce3 getOrDefault ["unwrapped", false, true];
+    private _turnedOn = _settingsAce3 getOrDefault ["turnedOn", false, true];
+    private _inUse = _settingsAce3 getOrDefault ["inUse", false, true];
+
+    // ACE3 Carry, Drag and Load will only be available if the device is not connected via power/network, 
+    // not unfold/opened/expanded, not turned on
+    if (_powerConnected || _networkConnected || _unwrapped || _turnedOn || _inUse) then
```

Comment:
Wouldn't be it better, if this is only triggered, if the status is changed?
Concider the case, if a laptop is already connected to power, so everything is already disabled.
So this does not have to run, if it is then connected to a network.
This applies to the hole function.

Reply:
I will introduce another veriable holding the current `_isBlocked` status. So blocking and unblocking is only done if it's a change of state.